src/types.rs

```diff
@@ -47,6 +47,8 @@ pub struct LanguageConfig {
     pub command: String,
     #[serde(default)]
     pub args: Vec<String>,
+    #[serde(default)]
+    pub envs: HashMap<String, String>,
```

Comment:
In hindsight, it would probably have been better to give users a more natural way to run shell commands. Right now we have

```toml
command = "sh"
args = ["-c", """CLASSPATH=... kotlin-language-server"""]
```

it could be as simple as

```toml
command = """CLASSPATH=... kotlin-language-server"""
```

Anyway, that would be a bigger change.
The envs addition looks good to me.

Reply:
FWIW I opened #608 so that this is tracked somewhere, since I noticed it didn't have an issue already.